### PR TITLE
Update README for strong-pm.io

### PR DESCRIPTION
**Pull request by sam-github**
See [README](https://github.com/strongloop/strong-pm/blob/readme-update/README.md)

Connect to strongloop/strongloop#191

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,273 +1,116 @@
...
+- Manage processes and cluster (use all the cores on your server), adjust sizes, built-in load balancer
+- Automatic restarts keep applications running forever
+- Zero-downtime redeploy/upgrade
+- View performance metrics on your application
+- View CPU profiles and heap snapshots to optimize performance and diagnose memory leaks
+- Use graphical tool StrongLoop Arc, **slc arc**, or command-line tool, **slc**
 
-Install [strong-build](http://github.com/strongloop/strong-build) and
-[strong-deploy](http://github.com/strongloop/strong-deploy):
+*** XXX We should put the comparison table right here methinks ***
```
Comment (sam-github) [inline README.md]: @kraman Will there be a text version of the table?

File: README.md [hunk 2]
```diff
@@ -1,273 +1,116 @@
...
-    cd loopback-example-app
-    sl-build
-    sl-deploy
+### Profile
 
-## Life-cycle
+- Heap snapshots and CPU profiles.
+- Profile local or remote applications with Arc.
+- Profile local applications with slc.
+- Use the _watchdog_ to start CPU profiling when the Node event loop stalls. ***XXX we should really remove this feature and only make it available through Arc... different issue though... maybe just remove for now.***
```
Comment (sam-github) [inline README.md]: @ijroth watchdog requires a *specific* watchdog license, even when used from CLI, isn't that enough?

File: README.md [hunk 3]
```diff
@@ -1,273 +1,116 @@
...
+### Metrics
 
-After preparation, the application is run.
+- View performance metrics in Arc
+- Use your own StatsD console if you prefer.
 
-## Installing as a Service
+***XXX I think above should enumerate the 3rdparty integrations both here and on
+strong-pm.io, I'll resync with strong-pm.io after Rand has updated it, see
+original Feature list - agree --Issac***
```
Comment (sam-github) [inline README.md]: @crandmck When you have text for this, please ping me, and I'll sync back up against the web site for the feature list.

Or you can edit this file in place, if you prefer.

File: README.md [hunk 4]
```diff
@@ -1,273 +1,119 @@
...
+- Manage processes and cluster (use all the cores on your server), adjust sizes, built-in load balancer
+- Automatic restarts keep applications running forever
+- Zero-downtime redeploy/upgrade
+- View performance metrics on your application
+- View CPU profiles and heap snapshots to optimize performance and diagnose memory leaks
+- Use graphical tool StrongLoop Arc, **slc arc**, or command-line tool, **slc**
 
-Install [strong-build](http://github.com/strongloop/strong-build) and
-[strong-deploy](http://github.com/strongloop/strong-deploy):
+***XXX We should put the comparison table right here methinks***
```
Comment (sam-github) [inline README.md]: @kraman will there be a text table version?

File: README.md [hunk 5]
```diff
@@ -1,273 +1,119 @@
...
-    sl-deploy
+### Profile
 
-## Life-cycle
+- Heap snapshots and CPU profiles.
+- Profile local or remote applications with Arc.
+- Profile local applications with slc.
+- Use the _watchdog_ to start CPU profiling when the Node event loop stalls.
+  ***XXX we should really remove this feature and only make it available through
+  Arc... different issue though... maybe just remove for now.***
```
Comment (sam-github) [inline README.md]: @ijroth watchdog requires a specific watchdog license, should be good enough, shouldn't it?

File: README.md [hunk 6]
```diff
@@ -1,273 +1,119 @@
...
-## Installing as a Service
+- View performance metrics in Arc
+- Use your own StatsD console if you prefer.
 
-The process manager should be installed as a service, so it gets integration
-with the system process manager. This will ensure it is started on machine boot,
-logs are correctly aggregated, permissions are set correctly, etc.
+***XXX I think above should enumerate the 3rdparty integrations both here and on
+strong-pm.io, I'll resync with strong-pm.io after Rand has updated it, see
+original Feature list - agree --Issac***
```
Comment (sam-github) [inline README.md]: @crandmck ping me when you have updated text on the strong-pm.io about page, or edit this in place if you want.

File: README.md [hunk 7]
```diff
@@ -1,273 +1,119 @@
...
 
-For more information on Docker and Docker Hub, see https://www.docker.com/
+- [StrongLoop Documentation](http://docs.strongloop.com/display/SLC/Operating+Node+applications)
+- [Professional support](http://strongloop.com/node-js/subscription-plans/)
+- [StrongLoop Google Group](https://groups.google.com/forum/#!forum/strongloop)
+- [GitHub issues](https://github.com/strongloop/strong-pm/issues)
+- [Gitter chat](https://gitter.im/strongloop/chat)
+- [Install on production server](./INSTALL.md)
+- [Application life-cycle](./LIFE-CYCLE.md)
+- ***XXX pointers to our blogs***
```
Comment (sam-github) [inline README.md]: I'll fill in when we have them.

File: README.md (hunk 4 above)
Comment (kraman) [inline README.md]: Don't have one at the moment. But can work with @crandmck to make one.

**Comment by sam-github** [comment]
This absorbs all suggestions up to now, including link for comparison, etc.

Only variance is instead of trying to keep the README and strong-pm.io both up-to-date as new blogs come out, I suggest that blogs get added to https://github.com/strongloop/strong-pm.io/blob/master/resources/index.html#L91

First two blogs are:
- https://strongloop.com/strongblog/node-js-deploy-production-best-practice/
- https://strongloop.com/strongblog/automatic-node-js-clustering-with-log-aggregation/ (though latter hasn't been updated for strong-pm yet: https://github.com/strongloop-internal/scrum-nodeops/issues/397)

Also, Loopback's README had these two links:
- [Professional support](http://strongloop.com/node-js/subscription-plans/)  
- [GitHub issues](https://github.com/strongloop/strong-pm/issues) 

but strong-pm.io doesn't, perhaps this is intentional, not sure, @ijroth?

Also... I'd prefer to remove duplicate links from the README docs and community, keep the list quite short there, and just point at strong-pm.io/resources: keeping two locations up-to-date is going to be painful, might as well focus on what is highest visibility (I assume), the website.

Thoughts, @ijroth?

Would like to merge this by tomorrow, if its looking OK.

**Comment by ijroth** [comment]
README mentions docker support but the comparison chart doesn't. Do we have it?

**Comment by ijroth** [comment]
I made some minor updates to the first section. 

I'm not sure what you mean by removing duplicate links, but I find the deep-linking we have now to be nice and I don't think those links will be changing much.

**Comment by sam-github** [comment]
@ijroth we have docker support: `curl -s0 http://strong-pm.io/docker.sh | sudo /bin/sh` will install from docker hub. its in the production page. we don't run apps inside a container... but we allow installation of strong-pm from docker, and have the above convenient script.

@kraman should we add into feature comparison matrix on strong-pm.io, and maybe your original in-depth google doc?

By deep linking, I mean adding links to all the blogs that are about to come out to two places, strong-pm.io and this README. But lets leave as is for now.

Anyhow, I'm OK with README as-is. Can I merge?

@ijroth ?

**Comment by sam-github** [comment]
And @ijroth (/cc @crandmck) ... the text changed in https://github.com/strongloop/strong-pm/commit/769a1b69242b0480be40b5700bbea560222f3605 was originally copied verbatim from https://github.com/strongloop/strong-pm.io/blob/master/index.html#L79-L97 ... perhaps you want Rand to change it there, too, for strong-pm.io?

**Comment by chandadharap** [comment]
Sam Please merge. More updates can be made as and when we need to.

**Pull request merged**